doc/customizing.rst

```diff
@@ -22,6 +22,15 @@ For information on how to configure your JupyterHub deployment, see the
 `JupyterHub for Kubernetes Customization Guide
 <https://zero-to-jupyterhub.readthedocs.io/en/latest/#customization-guide>`_.
 
+If you want to customise the spawner you can subclass it in ``extraConfig``.
```

Comment:
I tried to write a short explanation of what the reason for having 10-, 20- etc is. What do you think?

The keys (like `10-binder-customisations`) in the `extraConfig` dictionary will be merged
in alphabetical order. This means that you can use configuration from earlier keys in later ones.
In this case our custom spawner inherits from `BinderSpawner` which is defined in `00-binder`

Something like that?